File: programers_Lv1/x만큼 간격이 있는 n개의 숫자.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12954?language=javascript
function solution(x, n) {
  let addX = 0;
  const answer = Array.from({ length: n }, (_, i) => {
    addX += x;
    return addX;
  });
  return answer;
}

console.log(solution(2, 5));

File: programers_Lv1/x만큼 간격이 있는 n개의 숫자.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12954?language=python3
def solution(x, n):
    answer = [x]
    new_x = x
    for i in range(1, n):
        new_x += x
        answer.append(new_x)
    return answer


print(solution(2, 5))
